portal-api/api-tests/v1/gef/address.api-test.js

```diff
@@ -0,0 +1,30 @@
+const app = require('../../../src/createApp');
+const { MAKER, DATA_ADMIN, ADMIN } = require('../../../src/v1/roles/roles');
+const testUserCache = require('../../api-test-users');
+const { withClientAuthenticationTests } = require('../../common-tests/client-authentication-tests');
+const { withRoleAuthorisationTests } = require('../../common-tests/role-authorisation-tests');
+
+const { as, get } = require('../../api')(app);
+
+// TODO DTFS2-6626: this sends out real API calls to an external API - do we want to prevent this?
```

Comment:
Flagging question

Reply:
I've refactored `externalApi.controller.js` to be able to mock it, and I've mocked it